networking/ipv6.md

```diff
@@ -152,66 +141,31 @@ then the following changes will ensure IPv6 operation.
   ```
   {: .no-select-button}
 
-## Enabling IPv6 with OpenStack
-
-### Prerequisites
-
-When using {{site.prodname}} with a VM platform (e.g., OpenStack), obtaining IPv6
-connectivity requires certain configuration in the guest VM image:
-
--  When it boots up, the VM should issue a DHCPv6 request for each of
-   its interfaces, so that it can learn the IPv6 addresses that
-   OpenStack has allocated for it.
--  The VM must be configured to accept router advertisements.
--  If the VM uses the widely deployed DHCP client from ISC, it must
-   have a fix or workaround for {% include open-new-window.html text='this known
-   issue' url='https://kb.isc.org/article/AA-01141/31/How-to-workaround-IPv6-prefix-length-issues-with-ISC-DHCP-clients.html' %}.
-
-These requirements are not yet all met in common cloud images—but it
-is easy to remedy that by launching an image, making appropriate changes
-to its configuration files, taking a snapshot, and then using that
-snapshot thereafter instead of the original image.
-
-For example, starting from an Ubuntu cloud image, the following
-changes will suffice to meet the requirements just listed.
-
--   In `/etc/network/interfaces.d/eth0.cfg`, add:
-
-        iface eth0 inet6 dhcp
-                accept_ra 1
-
--   In `/sbin/dhclient-script`, add at the start of the script:
-
-        new_ip6_prefixlen=128
-
--   In `/etc/sysctl.d`, create a file named `30-eth0-rs-delay.conf` with
-    contents:
-
-        net.ipv6.conf.eth0.router_solicitation_delay = 10
-
-For CentOS, these additions to a cloud-init script have been reported to be effective:
-
-	runcmd:
-	- sed -i -e '$a'"IPV6INIT=yes" /etc/sysconfig/network-scripts/ifcfg-eth0
-	- sed -i -e '$a'"DHCPV6C=yes" /etc/sysconfig/network-scripts/ifcfg-eth0
-	- sed -i '/PATH/i\new_ip6_prefixlen=128' /sbin/dhclient-script
-	- systemctl restart network
+#### Enable dual stack, OpenShift
 
+TBD - operator steps  
 
-### Enabling IPv6 support in {{site.prodname}}
+#### Enable IPv6-only, OpenShift
 
-In OpenStack, IPv6 connectivity requires defining an IPv6 subnet, in
-each Neutron network, with:
+TBD - operator steps  
 
--   the IPv6 address range that you want your VMs to use
--   DHCP enabled
--   IPv6 address mode set to DHCPv6 stateful.
+#### Configure the Kubernetes control plane to operate over IPv6
```

Comment:
I think we agreed to move this material to a separate page.  Note that the rows that I've marked below as pertaining to workloads are covered by https://kubernetes.io/docs/concepts/services-networking/dual-stack/#prerequisites, which we've already linked to above.

Reply:
fixed